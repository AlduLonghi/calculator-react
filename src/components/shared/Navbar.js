import { Link } from 'react-router-dom';
import '../../style/App.css';

const Navbar = () => (
  <div>
    <h1>
      Math magicians
    </h1>
    <div>
      <Link to="/">Home</Link>
      <Link to="/calculator">Calculator</Link>
      <Link to="/quote">Quote</Link>
    </div>
  </div>
);

export default Navbar;
