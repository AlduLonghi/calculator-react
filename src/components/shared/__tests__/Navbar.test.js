import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from '../Navbar';

it('Snapshot test for Navbar component', () => {
    const tree = renderer
    .create(<Navbar />)
    .toJSON();
  expect(tree).toMatchSnapshot();
})