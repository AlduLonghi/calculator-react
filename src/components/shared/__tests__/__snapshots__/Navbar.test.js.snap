// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for Navbar component 1`] = `
<nav
  className="d-flex"
>
  <h1>
    Math Magicians
  </h1>
  <div
    className="nav-links d-flex"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      href="/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </div>
</nav>
`;
